airflow/api_fastapi/execution_api/routes/task_instances.py

```diff
@@ -30,14 +30,16 @@
 from airflow.api_fastapi.common.db.common import get_session
 from airflow.api_fastapi.common.router import AirflowRouter
 from airflow.api_fastapi.execution_api.datamodels.taskinstance import (
+    TIDeferredStatePayload,
     TIEnterRunningPayload,
     TIHeartbeatInfo,
     TIStateUpdate,
     TITerminalStatePayload,
 )
+from airflow.models import Trigger
 from airflow.models.taskinstance import TaskInstance as TI
 from airflow.utils import timezone
-from airflow.utils.state import State
+from airflow.utils.state import State, TaskInstanceState
```

Comment:
Since we already have State, we don't need to import `TaskInstanceState` too

Reply:
Handled it